### PR TITLE
Feature selector api

### PR DESCRIPTION
**Pull request by mollybsmith-noaa**
This pull request contains my API for getting scorecard metadata from individual apps. It uses server-side routes which are in the linked matsMethods.js commits. Documentation fo the API is at https://docs.google.com/document/d/1nX_hB0P9VyUokSYxWwtHwmB6MrTtklX0ZFlgOUtkyEg/edit?usp=sharing

### CODE REVIEW

**Review by randytpierce — APPROVED** [review]
I think this all looks good.

File: apps/precipGauge/server/main.js [hunk 1]
```diff
@@ -384,22 +382,6 @@ const doCurveParams = function () {
...
-                controlButtonCovered: true,
-                default: variables[0],
-                unique: false,
-                controlButtonVisibility: 'none',
-                displayOrder: 2,
-                displayPriority: 1,
-                displayGroup: 1
-            });
-    }
-
```
Comment (ian-noaa) [inline apps/precipGauge/server/main.js:402]: Just curiosity - were these moved elsewhere? I wasn't finding a place they were refactored to so I'm wondering if they were vestigial code.

**Review by ian-noaa — APPROVED** [review]
Looks great! Thanks for the commit structure in this PR, Molly. It certainly made it easier to review!

**Comment by mollybsmith-noaa** [comment]
No, they shouldn’t have been there in the first place. I added them
previously thinking they’d make an eventual scorecard easier, when in fact
they did the opposite, so I just removed them (those are single-variable
apps). Thanks for reviewing the pull request!

File: apps/precipGauge/server/main.js (hunk 1 above)
Comment (mollybsmith-noaa) [reply apps/precipGauge/server/main.js:402]: Well, my reply did not go to the right thread. Sigh. Here it is again in the correct place for posterity:

"No, they shouldn’t have been there in the first place. I added them
previously thinking they’d make an eventual scorecard easier, when in fact
they did the opposite, so I just removed them (those are single-variable
apps). Thanks for reviewing the pull request!"

Comment (mollybsmith-noaa) [reply apps/precipGauge/server/main.js:402]: Basically, it was code added specifically for the scorecard which the scorecard did not end up needing.

Comment (ian-noaa) [reply apps/precipGauge/server/main.js:402]: Makes sense. Thanks!

**Pull request merged**